genie-agent/src/main/java/com/netflix/genie/agent/execution/statemachine/actions/MonitorJobAction.java

```diff
@@ -139,6 +139,11 @@ protected Events executeStateAction(final ExecutionContext executionContext) {
         return Events.MONITOR_JOB_COMPLETE;
     }
 
+    @Override
```

Comment:
There must be a setting in your IDE that re-sorts methods and i don't have. I'm interested in finding out what it is so we can avoid this kind of changes in unrelated commits.

Reply:
Perhaps it's having the checkstyle plugin tied to our checkstyle file? Not sure if that impacts it but this looks like some rule that moves based on the scope of the methods and ordering. It didn't change when I did it via the default IDE reformat from the previous PR which I had included code cleanup checked. 🤷‍♂